app/src/main/java/org/mifos/mobile/ui/fragments/SavingAccountsTransactionFragment.kt

```diff
@@ -265,10 +264,10 @@ class SavingAccountsTransactionFragment : BaseFragment(), SavingAccountsTransact
         val llcheckBoxPeriod = dialogView?.findViewById<LinearLayout>(R.id.ll_row_checkbox)
         val checkBoxPeriod: AppCompatCheckBox? = dialogView?.findViewById(R.id.cb_select)
         val radioGroupFilter = dialogView?.findViewById<RadioGroup>(R.id.rg_date_filter)
+        val dateLayout = dialogView?.findViewById<LinearLayout>(R.id.start_end_date_layout)
```

Comment:
convert layout `id` in camelcase and use it as variable name

Reply:
@miPlodder please review changes